### PR TITLE
add tests for edit evaluation administrations screen

### PR DESCRIPTION
**Pull request by emmaacha**

### CODE REVIEW

**Review by Dev-Nino — APPROVED** [review]

File: __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+          name: "Evaluation 1",
+          eval_schedule_start_date: "2024-01-01T00:00:00.000Z",
+          eval_schedule_end_date: "2024-01-03T00:00:00.000Z",
+          eval_period_start_date: "2023-01-01T00:00:00.000Z",
+          eval_period_end_date: "2023-12-31T00:00:00.000Z",
+          remarks: "Remarks 1",
+          email_subject: "Subject 1",
+          email_content: "Content 1",
+          status: "Pending",
+          evaluation_results: [
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts:63]: not needed 🤔

File: __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+            },
+          ],
+        }),
+      })
+
+      if (isMobile) {
+        await page.getByTestId("SidebarCloseButton").click()
+      }
+
+      await expect(page.getByPlaceholder("Evaluation name")).toBeVisible()
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts:117]: should check the input value

File: __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts [hunk 3]
```diff
@@ -0,0 +1,687 @@
...
+
+      await expect(page.getByLabel("Period (from)")).toBeVisible()
+      await expect(page.getByLabel("Period (to)")).toBeVisible()
+      await expect(page.getByLabel("Schedule (from)")).toBeVisible()
+      await expect(page.getByLabel("Schedule (to)")).toBeVisible()
+
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts:123]: same

File: __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts [hunk 4]
```diff
@@ -0,0 +1,687 @@
...
+      await expect(
+        page.getByLabel("Evaluation description/notes")
+      ).toBeVisible()
+
+      await expect(page.getByPlaceholder("Subject")).toBeVisible()
+      await expect(
+        page.getByRole("textbox", { name: "Some description" })
+      ).toBeVisible()
+
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts:132]: same

File: __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts [hunk 5]
```diff
@@ -0,0 +1,687 @@
...
+          name: "Evaluation 1",
+          eval_schedule_start_date: "2024-01-01T00:00:00.000Z",
+          eval_schedule_end_date: "2024-01-03T00:00:00.000Z",
+          eval_period_start_date: "2023-01-01T00:00:00.000Z",
+          eval_period_end_date: "2023-12-31T00:00:00.000Z",
+          remarks: "Remarks 1",
+          email_subject: "Subject 1",
+          email_content: "Content 1",
+          status: "Pending",
+          evaluation_results: [
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts:177]: not needed

File: __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts [hunk 6]
```diff
@@ -0,0 +1,687 @@
...
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts:276]: same

File: __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts [hunk 7]
```diff
@@ -0,0 +1,687 @@
...
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts:434]: same

File: __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts [hunk 8]
```diff
@@ -0,0 +1,687 @@
...
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/[id]/edit/edit_evaluation.spec.ts:538]: same

**Review by jatnerubia — COMMENTED** [review]
just a few changes

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes

**Review by jatnerubia — APPROVED** [review]
Nice

**Pull request merged**